src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/IdeTestFrameworkExecutor.cs

```diff
@@ -19,15 +19,9 @@ public IdeTestFrameworkExecutor(AssemblyName assemblyName, ISourceInformationPro
         [SuppressMessage("Usage", "VSTHRD100:Avoid async void methods", Justification = "Follows pattern expected by Xunit framework.")]
         protected override async void RunTestCases(IEnumerable<IXunitTestCase> testCases, IMessageSink executionMessageSink, ITestFrameworkExecutionOptions executionOptions)
         {
-            try
-            {
-                using (var assemblyRunner = new IdeTestAssemblyRunner(TestAssembly, testCases, DiagnosticMessageSink, executionMessageSink, executionOptions))
-                {
-                    await assemblyRunner.RunAsync();
-                }
-            }
-            catch
+            using (var assemblyRunner = new IdeTestAssemblyRunner(TestAssembly, testCases, DiagnosticMessageSink, executionMessageSink, executionOptions))
             {
+                await assemblyRunner.RunAsync();
```

Comment:
There is a bunch of exception handling code in places below this to report test failures if the harness fails.  

However the exception handling code attempts to report actual test case failures via the xunit API.  If those throw, then the exception bubbles all the way here, and would previously just silently catch it (though there are some logs that contain the exception).

Now, if we get here the actual test runner reports an unhandled exception and doesn't succeed.

This also matches what the xunit samples do, e.g. https://github.com/xunit/samples.xunit/blob/28d3683f74b104d33544efe5d1ae45ce9b0ad8c5/v2/AssemblyFixtureExample/XunitExtensions/XunitTestFrameworkExecutorWithAssemblyFixture.cs#L14